src/IconMenu/IconMenu.spec.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env mocha */
+import React from 'react';
+import {shallow} from 'enzyme';
+import {assert} from 'chai';
+import getMuiTheme from '../styles/getMuiTheme';
+
+import IconMenu from './IconMenu';
+
+describe('<IconMenu />', function() {
+  const muiTheme = getMuiTheme();
+  const shallowWithContext = (node) => shallow(node, {context: {muiTheme}});
+  it('add iconStyle if "iconButtonElement.props.iconStyles" is truthy', () => {
```

Comment:
Won't this line throw some warning in the console as `iconStyle` is unknow?
Also, the deprecating warning we have added is going to throw too.
I think that we can remove this test as we want to get ride of the `iconStyle` prop. I'm not too worry about having a regression here.